rllib/agents/ddpg/ddpg_torch_policy.py

```diff
@@ -63,11 +64,20 @@ def ddpg_actor_critic_loss(policy, model, _, train_batch):
         clipped_normal_sample = torch.clamp(
             torch.normal(
                 mean=torch.zeros(policy_tp1.size()),
-                std=policy.config["target_noise"]), -target_noise_clip,
-            target_noise_clip)
-        policy_tp1_smoothed = torch.clamp(policy_tp1 + clipped_normal_sample,
-                                          policy.action_space.low.item(0),
-                                          policy.action_space.high.item(0))
+                std=policy.config["target_noise"]).to(policy_tp1.device),
+            -target_noise_clip, target_noise_clip)
```

Comment:
`to(policy_tp1.device)` is still missing

Reply:
Clamp returns a tensor on the same device, so it's already on the correct device.

Reply:
Yes, I was talking about the output of `torch.normal`. Maybe I missed it when I wrote this comment. Anyway it is fine in the current commit.